api_testing/apis/test_plan_api.py

```diff
@@ -315,16 +315,19 @@ def create_testplan_with_article_not_all(self):
         else:
             raise Exception(f'cant create the test plan with payload {payload}')
 
-    def create_completed_testplan_random_data(self):
+    def create_completed_testplan_random_data(self,no_testplans=1):
```

Comment:
could you rename `no_testplans` to `no_testunits` to make name clear, you create only one test plan with multiple test units

Reply:
done